galleries/examples/subplots_axes_and_figures/subplots_demo.py

```diff
@@ -213,8 +213,7 @@
 # %%
 # .. tags::
 #
-#    component: subplot
-#    plot-type: line
-#    plot-type: polar
+#    component: subplot, component: axes, component: axis
```

Comment:
Can we generally agree to only use one tag per line? I find it hard to read multiple tags when the tags contain spaces.

Reply:
I still believe 15 lines tags is better than a block of 5 lines with 3 tags each. And yes, long lists may push in the direction of not using too many tags, which IMHO is a good thing 😄 .